### ISSUE TITLE
Question: Dealing with slice of structs

### ISSUE THREAD

**Issue by 4ydx** [gopherjs/gopherjs#631]
https://gopherjs.github.io/playground/#/R6EvY6PsPa

How can this be done?  If I do not include the *js.Object in the "Value" struct, gopherjs complains about not being able to internalize Value.  Including *js.Object in Value runs, but the values (10) are not available in the window.Test object that results.

You will need to open the browser's javascript console and inspect window.Test to see this of course.

**Comment by flimzy** [comment]
I see an empty playground when I click on that link.

**Comment by 4ydx** [comment]
Works for me :P

**Comment by flimzy** [comment]
it's working for me now.. *shrug*

**Comment by 4ydx** [comment]
Man I see what I did wrong.  I am enjoying being able to use golang to write javascript but some of the requirements keep tripping me up :package: 

The following is properly edited to work.  I was forgetting to add the `js:"<label>"` tag to the struct that makes up elements in the array.

https://gopherjs.github.io/playground/#/XXls6koTW0

**Comment by myitcv** [comment]
@4ydx glad you worked out what was going wrong.

The requirement for `js` tags is being discussed in a couple of open issues #236 and #625

**Comment by shurcooL** [comment]
Just checking, but did you @4ydx read over https://github.com/gopherjs/gopherjs/wiki/JavaScript-Tips-and-Gotchas? Is this situation not covered there?

**Comment by 4ydx** [comment]
I did, but didn't understand that structs within structs, however that may come about, require that they all have *js.Object in them.  So initially I only had a top level *js.Object.  I realized that probably wasn't correct and added *js.Object to the struct defining the values in the slice.  It wasn't until I was out cycling that I realized I had probably left off the js:"<tag>" value.  So it is all on me.  Apologies for creating these questions that are probably little more than clutter.  Maybe this will finally cement it for me.
